src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsintegrationapi/smoke/PersonSmokeTest.kt

```diff
@@ -14,25 +14,53 @@ class PersonSmokeTest : DescribeSpec({
   val baseUrl = "http://localhost:8080"
   val httpClient = HttpClient.newBuilder().build()
   val httpRequest = HttpRequest.newBuilder()
-  val id = "A1234AL"
 
-  it("returns a person from NOMIS, Prisoner Offender Search and Probation Offender Search") {
+  it("returns a list of persons using first name and last name as search parameters") {
+    val firstName = "Example_First_Name"
+    val lastName = "Example_Last_Name"
+    val queryParams = "firstName=$firstName&lastName=$lastName"
+
     val response = httpClient.send(
-      httpRequest.uri(URI.create("$baseUrl/persons/$id")).build(),
+      httpRequest.uri(URI.create("$baseUrl/persons?$queryParams")).build(),
       HttpResponse.BodyHandlers.ofString()
     )
 
-    response.body().shouldContain("\"nomis\":{\"firstName\":\"string\",\"lastName\":\"string\"")
+    response.statusCode().shouldBe(HttpStatus.OK.value())
+    response.body().shouldContain("\"persons\":[")
+    response.body().shouldContain(
+      """
+        "firstName":"Robert",
+        "lastName":"Larsen"
+      """.removeWhitespaceAndNewlines()
+    )
+    response.body().shouldContain(
+      """
+        "firstName":"string",
+        "lastName":"string"
+      """.removeWhitespaceAndNewlines()
+    )
   }
 
-  it("returns image metadata for a person") {
-    val response = httpClient.send(
-      httpRequest.uri(URI.create("$baseUrl/persons/$id/images")).build(),
-      HttpResponse.BodyHandlers.ofString()
-    )
+  describe("returns a person from NOMIS, Prisoner Offender Search and Probation Offender Search when given an id") {
```

Comment:
To me this describe is unnecessary but do as you will!

Reply:
Yeeah fair comment. Let's bin it, it just adds confusion